frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -473,11 +417,87 @@ public void refreshModel() {
   }
 
   @UIEffect
-  private void initColumns() {
-    createDefaultColumnsFromModel();
+  private void refreshModel(
```

Comment:
Could you move the methods back to their original place? so that the actual changes are easier to see